wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -155,12 +155,17 @@ private fun parseFileDescriptor(fileDescriptor: DescriptorProtos.FileDescriptorP
 private fun parseEnum(path: List<Int>, helper: SourceCodeHelper, enum: DescriptorProtos.EnumDescriptorProto, descs: DescriptorSource): EnumElement {
   val info = helper.getLocation(path)
   val constants = mutableListOf<EnumConstantElement>()
-  for (enumValueDescriptorProto in enum.valueList) {
+  for ((index, enumValueDescriptorProto) in enum.valueList.withIndex()) {
+    val enumPaths = mutableListOf(*path.toTypedArray())
+    enumPaths.add(DescriptorProtos.EnumDescriptorProto.VALUE_FIELD_NUMBER)
+    enumPaths.add(0)
```

Comment:
I would expect this to be outside the loop.

If you are reconstructing the entire path every single time, you don't even need the `add(0)` then setting to `index`, but could just do `enumPaths.add(index)`